parsec/core/fs/fs.py

```diff
@@ -194,12 +214,12 @@ def _get_workspace(self, path):
         cooked_path = FsPath(path)
         if not cooked_path.is_workspace():
             # Will fail with correct exception
-            workpace_name = path
+            workspace_name = path
```

Comment:
Damn typo, it has reproduce like cockroaches !!!!

Reply:
Yea I noticed this fix in your PR too ^^